velox/exec/Task.cpp

```diff
@@ -1676,9 +1677,13 @@ void Task::setError(const std::string& message) {
   }
 }
 
+std::string Task::safeErrorMessage() const {
```

Comment:
Usually we name this method as errorMessageLocked instead of safeErrorMessage. And the comment for safeErrorMessage is not necessary. Thanks!